docker/Dockerfile.debian

```diff
@@ -1,8 +1,23 @@
 FROM debian:buster-slim as builder
 WORKDIR /usr/src
 
+ENV PHPIZE_DEPS \
+		autoconf \
+		dpkg-dev \
+		file \
+		g++ \
+		gcc \
+		libc-dev \
+		make \
+		pkg-config \
+		re2c
+
 RUN apt-get update -y \
-  && apt-get install -y build-essential autoconf libtool bison re2c curl less pkg-config libxml2-dev libsqlite3-dev
+  && apt-get install -y \
+    ${PHPIZE_DEPS} \
+    curl \
+    libxml2-dev \
+    libsqlite3-dev
 
 ARG PHP_VERSION=8.1.10
```

Comment:
We should probably change this to use the same logic to find the most recent version.

Reply:
I didn't want to do this in the dockerfile (because it will screw with the github build), so I've made PHP_VERSION a required param and documented that.